integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/ReplaceKeys.feature

```diff
@@ -32,20 +32,18 @@ Feature: SmartMetering Configuration - Replace Keys
     And the new keys are stored in the secret management database encrypted_secret table
 
   @RecoverKeys
-  Scenario: Recover keys after a (simulated) failed key change
-    #Try to connect using incorrect (swapped) keys and then try to recover the correct new keys
+  Scenario: Recover keys after a (simulated) failed key change (incorrect E key)
```

Comment:
Should there not also be a scenario for an incorrect A key?

Reply:
I decided not to implement an integration test to only test an || :
https://github.com/OSGP/open-smart-grid-platform/pull/732/files#diff-69d5d4719c0214cb56df9ccf644aef39aff9cbb856157d20dba53ac6d5a602b4R333